assembly-main/src/assembly/bin/che.sh

```diff
@@ -462,11 +471,13 @@ function call_catalina {
     return
   fi
 
-  # Che requires Java version 1.8 or higher.
-  JAVA_VERSION=$("${JAVA_HOME}/bin/java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
-  if [[  -z "${JAVA_VERSION}" || "${JAVA_VERSION}" < "1.8" ]]; then
-      error_exit "Che requires Java version 1.8 or higher. We found a lower version."
-      return
+  if [[ "${SKIP_JAVA_VERSION}" == false ]]; then
+    # Che requires Java version 1.8 or higher.
+	JAVA_VERSION=$("${JAVA_HOME}/bin/java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+	if [[  -z "${JAVA_VERSION}" || "${JAVA_VERSION}" < "1.8" ]]; then
+	  error_exit "Che requires Java version 1.8 or higher. We found a lower version."
```

Comment:
It will be easier to find problems with java version if you put found java version in error message